### DISCUSSION TITLE
关于下载geoip.dat文件和geosite.dat文件

### DISCUSSION THREAD

**Discussion by ShaoShenYun** [v2rayA/v2rayA#1424] [Ideas]
我用snap安装完成后一直提示正在下载这两个文件，但因为在国内不能访问这个下载地址，于是就卡着页面进不去了，能否改成先不管这两个文件，等进去配置好了代理后再开始下载。

**Comment by xcz1456** [comment]
我也一直是这个问题，难道要先装好其他的代理软件后，才能下载这两个文件，才能正常使用这个代理软件？

**Comment by Mactarvish** [comment]
同问

**Comment by xcz1456** [comment]
试了下，docker可以用，而docker现在不能像以前那样直接拉取，要想别的办法拉取下来。
